tools/tests/test.config

```diff
@@ -5,5 +5,17 @@
       "query": "select * from time;",
       "interval": 1
     }
-  ]
+  ],
+  "threat_intel" : {
```

Comment:
Why pigeonhole yourself into "threats" or "intel", why not make this a map of "filesystem_watches"? Then you can determine/apply "threat/intel" after the fact, divorced from the alerting/monitoring config. 

If you want to make it SUPER robust you could make a "auto_events" or "subscriptions":

```
subscriptions: {
  downloads: {
    publisher: FSEvents,
    paths: ["/some/path", "/another/path"]
  },
[...]
}
```

Then when you pull in the JSON dict for "subscriptions" you introspect into the "event_publisher" type and call the appropriate `Registry::getRegistry("event_publisher", "<TYPE>")->autoSubscribe(your_json_here={})` then require each EventPublisher to create/subscribe using that prototype. In the case of "FSEvents", it would know to expect a "paths" vector. You'd have to take care of the TablePlugin callback bindings, but that must happen either way.

Mostly food for thought.